### PR TITLE
request ROS version in issue template

### PR DESCRIPTION
**Pull request by jpsim**
could be useful when narrowing down issues

### CODE REVIEW

**Comment by jpsim** [comment]
Bump @bdash

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -39,9 +39,11 @@ Full Xcode projects that we can compile ourselves are ideal!
 ## Version of Realm and Tooling
 <!---
 [In the CONTRIBUTING guidelines](https://git.io/vgxJO), you will find a script,
-which will help determining these versions.
+which will help determining some of these versions.
 -->
-Realm version: ?
+Realm Client version: ?
```
Comment (bdash) [inline .github/ISSUE_TEMPLATE.md]: I don't think we use the terminology "Realm Client" elsewhere?

Comment (jpsim) [inline .github/ISSUE_TEMPLATE.md]: What do you suggest?

Comment (jpsim) [inline .github/ISSUE_TEMPLATE.md]: "Realm Objective-C or Swift version"?

Comment (bdash) [inline .github/ISSUE_TEMPLATE.md]: Perhaps Realm framework version?

Comment (jpsim) [inline .github/ISSUE_TEMPLATE.md]: Done.

**Pull request merged**